recipes/vcboost/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.0.0" %} 
+
+package: 
+  name: vcboost 
+  version: {{ version }} 
+  
+source: 
+  url: https://github.com/oranges7/VCboost/archive/refs/heads/main.zip
+  md5: 43a113509e361c9f262115a74add49e8
+  sha1: 97070bb1487adc6e1b14642d99f1cfba99dd2ba7
+  sha256: e46d76702e92aba318d50169b64188f62aea5d5839d610cc196aadd8ac199073 
+  
+build: 
+  noarch: generic 
+  number: 0 
+  run_exports:
+  - vcboost 1.0.0  
```

Comment:
please look at the help in the PR description how this should look like

Reply:
Thank you for your reply.I've seen PR description, but I still don't understand it. For example, my program can only run on linux and needs to be described. And what exactly does run_exports mean?

Reply:
My error is as follows. Do you know why?Looking forward to your reply.

File "/opt/mambaforge/envs/bioconda/lib/python3.8/site-packages/ruamel/yaml/scanner.py", line 626, in fetch_value
raise ScannerError(ruamel.yaml.scanner.ScannerError: mapping values are not allowed here in "<unicode string>", line 2, column 8:
package:
             ^ (line: 2)